scipy/signal/tests/test_filter_design.py

```diff
@@ -813,8 +896,8 @@ def test_output_type(self):
 
 
 def dB(x):
-    # Return magnitude in decibels
-    return 20 * np.log10(abs(x))
+    # Return magnitude in decibels, avoiding divide-by-zero warnings
+    return 20 * np.log10(np.maximum(np.abs(x), 1e-100))
```

Comment:
I did that at one point but it ends up being more invasive, because some of the assertions (maybe `assert_array_less`?) didn't like `inf`s in there